components/mixins/profile/profile-internet-subscription.schema.json

```diff
@@ -0,0 +1,71 @@
+{
+  "meta:license": [
+    "Copyright 2020 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/mixins/profile/profile-internet-subscription",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Internet Subscription",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "meta:tags": { "industry": ["telecom"] },
+  "meta:intendedToExtend": [
+    "https://ns.adobe.com/xdm/mixins/profile/profile-telecom-subscription"
+  ],
+  "description": "Collects all specific details related to a customer's internet subscription plan, including data cap, connection type, and speed details.",
```

Comment:
Can we shorten the description where possible? Sorry I know its seems nit picky, but we just updated all XDM descriptions to be as short as possible so I'm trying to keep it consistent. Thanks!

"details related to an internet subscription plan, such as including data cap, connection type, and speed."